src/frontend.rs

```diff
@@ -41,9 +41,15 @@ impl SwayFrontend {
 					if saved.width > 0 && saved.height > 0 {
 						l += &format!(" resolution {}x{}", saved.width, saved.height);
 					}
+					let mut v = ["90".to_string(),"180".into(),"270".into(),"flipped".into(),"flipped-90".into(),"flipped-180".into(),"flipped-270".into()];
+					if v.contains(&saved.transform) {
```

Comment:
Maybe we can replace this with a `match`?

Reply:
Yes, it most certainly can.